SiVL19/get_iu_xray.py

```diff
@@ -117,4 +139,4 @@ def split_cases(reports_images, reports_text, keys, filename):
 test_path = "iu_xray/test_images.tsv"
 
 split_cases(reports_with_images, text_of_reports, train_keys, train_path)
-split_cases(reports_with_images, text_of_reports, test_keys, test_path)
\ No newline at end of file
+split_cases(reports_with_images, text_of_reports, test_keys, test_path)
```

Comment:
Should we also add split for the tags?

Reply:
I am not sure I am following your thought. How would that work exactly?

Reply:
The same way it works for the captions. We could create a dictionary tags_of_reports with the report as key and the tags as value and then call split_cases.

Reply:
I am still failing to see the need/benefits of this step.